.vscode/extensions.json

```diff
@@ -4,7 +4,8 @@
         "editorconfig.editorconfig",
         "github.vscode-github-actions",
         "ms-dotnettools.vscode-dotnet-runtime",
-        "ms-dotnettools.csdevkit"
+        "ms-dotnettools.csdevkit",
+        "alexcvzz.vscode-sqlite"
```

Comment:
This file is mixing tabs and spaces, pick one. (4 spaces should be the default according to our .editorconfig)